lib/dto/giphy_assets.dart

```diff
@@ -17,6 +30,10 @@ class GiphyAssets {
     this.size4k,
   });
 
+  /// Creates a [GiphyAssets] instance from a JSON object.
```

Comment:
just curious - do we have a use case for this?

Reply:
yup, we use it to convert video objects from the native sdks.